### PR TITLE
Update lib.rs

### PR DESCRIPTION
**Pull request by lunarforky-z**

### CODE REVIEW

File: waw/src/lib.rs [hunk 1]
```diff
@@ -3,7 +3,9 @@
 #![doc = include_str!("../../README.md")]
 #![warn(missing_docs)]
 
+
 /// Audio buffer structs
+
```
Comment (hylcore-V) [inline waw/src/lib.rs]: don't need to change this part of the code,
in the rust project,  there is no need to give the br in this part

**Review by hylcore-V — CHANGES_REQUESTED** [review]

File: waw/src/lib.rs (hunk 1 above)
Comment (lunarforky-z) [reply waw/src/lib.rs]: yes okay
i will change this

Comment (hylcore-V) [reply waw/src/lib.rs]: okay
this is solved

**Comment by lunarforky-z** [comment]
oh,  i can understand ur change in the code

**Comment by hylcore-V** [comment]
if this change can help the readable of our code and make sense of the rust code

**Comment by hylcore-V** [comment]
and now u can merge the pr

**Comment by lunarforky-z** [comment]
yes,  i got it.
thx

**Pull request merged**